src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -146,6 +146,7 @@ public static class Features
             public const string ExtractMethod = "Refactoring.ExtractMethod";
             public const string FindReferences = nameof(FindReferences);
             public const string F1Help = nameof(F1Help);
+            public const string ForeachToFor = nameof(ForeachToFor);
```

Comment:
Consider aligning name with Cyrus's refactoring from for to foreach:
`   public const string CodeActionsConvertForToForEach = "CodeActions.ConvertForToForEach";`

Reply:
Note: ConvertForEachToFor is definitely the more appropriate name.  it also fits with existing features we've already shipped (like ConvertIfToSwitch).